src/Form/FormControlDecoratorGroup.test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { FormFieldContext } from './FormFieldContext';
+import FormControlDecoratorGroup from './FormControlDecoratorGroup';
+import { FORM_CONTROL_SIZES } from './constants';
+
+describe('FormFieldDecoratorGroup', () => {
+  it('renders', () => {
+    const wrapper = mount((
+      <FormControlDecoratorGroup
+        leadingElement="before"
```

Comment:
I know you're still adding tests, but we should probably test different options for these?

Reply:
Added another test for rendering nodes instead of strings.